docs/controls/file-browser/v8-schema.md

```diff
@@ -89,200 +113,448 @@ This outlines the full schema available to configure the btowser. These options
                  */
                 fallbackToRoot?: boolean;
             };
+            /**
+             * Indicates that File Browser should start in the user's recent files.
+             */
+            recent?: {};
+            /**
+             * Indicates that File Browser should start in the files shared with the user.
+             */
+            sharedWithMe?: {};
+            /**
+             * Indicates that File Browser should start in the user's lists.
+             */
+            myLists?: {};
+            /**
+             * Indicates that File Browser should start in the user's recent lists.
+             */
+            recentLists?: {};
+        };
+        /**
```

Comment:
Let's not expose `shortcut` publicly. We never ultimately shipped anything which uses this.